eppo_core/src/ufc/assignment.rs

```diff
@@ -29,10 +31,38 @@ pub enum AssignmentValue {
     /// A boolean value.
     Boolean(bool),
     /// Arbitrary JSON value.
-    Json(Arc<serde_json::Value>),
+    Json {
+        raw: Str,
```

Comment:
Saving raw json value here. This will also allow us to return raw string in some SDKs if users want to use their own parser.